maps/away/voxship/voxship_jobs.dm

```diff
@@ -1,177 +1,187 @@
+	title = "Quill"
 	total_positions = 1
-	outfit_type = /decl/hierarchy/outfit/job/voxship/crew/captain
-	supervisors = "The Apex and the arkship"
-	info = "You're in charge. You fly the ship, and dictate what the crew does. You are in charge of keeping your subordinates in check, the Apex has given you authority to kill any that disobeys your orders. Do not disappoint the Apex."
-	whitelisted_species = list(SPECIES_VOX, SPECIES_VOX_ARMALIS)
-	min_skill = list(	SKILL_HAULING     = SKILL_BASIC,
-						SKILL_EVA         = SKILL_EXPERT,
-						SKILL_SCIENCE     = SKILL_BASIC,
-						SKILL_PILOT       = SKILL_ADEPT,
-						SKILL_COMBAT      = SKILL_ADEPT,
-						SKILL_WEAPONS     = SKILL_ADEPT)
-
-
-	max_skill = list(   SKILL_BUREAUCRACY = SKILL_EXPERT,
-	                    SKILL_FINANCE = SKILL_EXPERT,
-	                    SKILL_EVA = SKILL_SPEC,
-	                    SKILL_MECH = SKILL_EXPERT,
-	                    SKILL_PILOT = SKILL_SPEC,
-	                    SKILL_HAULING = SKILL_EXPERT,
-	                    SKILL_COMPUTER = SKILL_EXPERT,
-	                    SKILL_BOTANY = SKILL_EXPERT,
-	                    SKILL_COOKING = SKILL_EXPERT,
-	                    SKILL_COMBAT = SKILL_SPEC,
-	                    SKILL_WEAPONS = SKILL_SPEC,
-	                    SKILL_FORENSICS = SKILL_EXPERT,
-	                    SKILL_CONSTRUCTION = SKILL_EXPERT,
-	                    SKILL_ELECTRICAL = SKILL_EXPERT,
-	                    SKILL_ATMOS = SKILL_EXPERT,
-	                    SKILL_ENGINES = SKILL_EXPERT,
-	                    SKILL_DEVICES = SKILL_EXPERT,
-	                    SKILL_SCIENCE = SKILL_EXPERT,
-	                    SKILL_MEDICAL = SKILL_EXPERT,
-	                    SKILL_ANATOMY = SKILL_EXPERT,
-	                    SKILL_CHEMISTRY = SKILL_EXPERT)
-	skill_points = 30
-
-#define VOXSHIP_OUTFIT_JOB_NAME(job_name) ("Vox Asteroid Base - Job - " + job_name)
+	outfit_type = /decl/hierarchy/outfit/job/voxship/crew
+	supervisors = "apex and the arkship"
+	info = "You're in charge. You fly the ship, and dictate what the crew does. Do not disappoint the Apex."
+	min_skill = list(
+		SKILL_HAULING = SKILL_BASIC,
+		SKILL_EVA = SKILL_EXPERT,
+		SKILL_SCIENCE = SKILL_ADEPT,
+		SKILL_PILOT = SKILL_ADEPT,
+		SKILL_COMBAT = SKILL_ADEPT,
+		SKILL_WEAPONS = SKILL_ADEPT
+	)
+
+	max_skill = list(
+		SKILL_BUREAUCRACY = SKILL_MAX,
+		SKILL_FINANCE = SKILL_MAX,
+		SKILL_EVA = SKILL_MAX,
+		SKILL_MECH = SKILL_MAX,
+		SKILL_PILOT = SKILL_MAX,
+		SKILL_HAULING = SKILL_MAX,
+		SKILL_COMPUTER = SKILL_MAX,
+		SKILL_BOTANY = SKILL_MAX,
+		SKILL_COOKING = SKILL_MAX,
+		SKILL_COMBAT = SKILL_MAX,
+		SKILL_WEAPONS = SKILL_MAX,
+		SKILL_FORENSICS = SKILL_MAX,
+		SKILL_CONSTRUCTION = SKILL_MAX,
+		SKILL_ELECTRICAL = SKILL_EXPERT,
+		SKILL_ATMOS = SKILL_EXPERT,
+		SKILL_ENGINES = SKILL_EXPERT,
+		SKILL_DEVICES = SKILL_MAX,
+		SKILL_SCIENCE = SKILL_MAX,
+		SKILL_MEDICAL = SKILL_EXPERT,
+		SKILL_ANATOMY = SKILL_EXPERT,
+		SKILL_CHEMISTRY = SKILL_EXPERT
```

Comment:
Not sure if raising almost all the Vox skillcaps to max is the way to go...

Reply:
That's intentional, since they are supposed to be more of a "strong off-ship raider role" and already got nerfs to the damage they can take.